.travis.yml

```diff
@@ -12,13 +12,10 @@ branches:
     - travis
 
 rvm:
-  - 1.8.7
```

Comment:
Some tests might have failed, but I'd like to keep 1.8 compatibility for the _low level_  gems (transport+api) as long as technically possible. (The Rails integration and the `elasticsearch-dsl` gem are already 1.9+ only.)